include/exadg/operators/inverse_mass_operator.h

```diff
@@ -186,6 +187,38 @@ class InverseMassOperator
     }
   }
 
+  // dst = scaling_factor * (M^-1 * src)
+  void
+  apply_scale(VectorType & dst, double const scaling_factor, VectorType const & src) const
+  {
+    dst.zero_out_ghost_values();
```

Comment:
In the else-branch below, the function `apply()` is called, which internally calls `dst.zero_out_ghost_values();`. So I think we should shift this line in order to avoid misunderstandings and/or redundant function calls. Additionally, a comment why we need this line would be helpful in my opinion.